packages/auth/src/shared/utils.ts

```diff
@@ -13,3 +14,7 @@ export function accessToBody(access: Access) {
     )
   )
 }
+
+export function generateNonceOrToken(): string {
+  return crypto.randomBytes(8).toString('hex').toUpperCase()
```

Comment:
While the GNAP spec only requires token value adhere to :point_down:, the example access tokens are larger than example nonce or continue tokens, and none of them appear to be hex-encoded
https://www.rfc-editor.org/rfc/rfc9110#section-11.2

Reply:
I use the hex encoding to convert into a series of alphanumeric characters, but the length is definitely longer so I'll make a version for tokens.

Reply:
GNAP Java appears to use:
https://github.com/bspk/oauth.xyz-java/blob/283c52f49d916e3b77a9d1744fd28d82f8c45af4/lib/src/main/java/io/bspk/oauth/xyz/data/AccessToken.java#L44-L45